src/components/Details/Flavor/Flavor.js

```diff
@@ -44,13 +44,14 @@ export class Flavor extends React.Component {
           template: result,
         });
       })
-      .catch(error => {
-        this.props.onLoadError(error.message || 'An error occurred while loading vm flavors. Please try again.');
-        return this.setState({
+      .catch(error =>
+        // An error occurred while loading template, it's ok, template may be deleted.
+        // Ref: BZ1685393
```

Comment:
is it necessary to include the BZ number?

Reply:
@rawagner , 

> is it necessary to include the BZ number?

No :-) , but it is strange not to do anything on error ... so I needed some excuse.
looking again at @rawagner comment https://github.com/kubevirt/web-ui-components/pull/287#issuecomment-476177652 it may be better to find some way to indicate that the template is missing ... so this will be replaced with actually doing something, and no need for the BZ ref.

Reply:
@suomiy sorry ^^